model/import/CsvImportForm.php

```diff
@@ -26,13 +26,18 @@
 use tao_helpers_form_FormFactory;
 use tao_helpers_form_xhtml_Form;
 use tao_models_classes_import_CsvUploadForm;
+use oat\oatbox\service\ServiceManager;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use oat\tao\helpers\UrlHelper;
 
 class CsvImportForm extends tao_models_classes_import_CsvUploadForm
 {
-    public function __construct(array $data = [], array $options = [])
-    {
-        $options[tao_models_classes_import_CsvUploadForm::IS_OPTION_FIRST_COLUMN_ENABLE] = false;
+    public $uri;
 
+    public function __construct(array $data = [], array $options = [])
+    {   
+        $this->uri = $options['classUri'];
```

Comment:
I suggest to be more defensive and prevent cases of incorrect usage, e.g. if someone later will not provide option you are relying on class at least shouldn't generate warnings

```suggestion
        $this->uri = $options['classUri'] ?? '';
```